Add documentation for GCP Service Classes (#1717)
kyma-project/kyma: docs/service-brokers/gcp-broker-service-classes/cloud-sql-mysql/docs/overview.md

---
title: Overview
type: Overview
---

**Cloud SQL-MySQL** is a fully-managed database service that makes it easy to set up, maintain, manage, and administer your relational MySQL databases in the cloud. The Cloud SQL offers high performance, scalability, and convenience. Hosted on Google Cloud Platform, Cloud SQL provides a database infrastructure for applications running anywhere.

For more details, read the **Plans Details** document.

> >**NOTE:** Google does not provide the ability to specify the broker's version so bear in mind that you use the most up-to-date version of the GCP Broker.

Reviewer: ```suggestion
>**NOTE:** Google cannot provide the ability to specify the broker's version so bear in mind that you use the most up-to-date version of the GCP Broker.
```
